query/stdlib/experimental/to.go

```diff
@@ -37,7 +36,6 @@ type ToOpSpec struct {
 func init() {
 	toSignature := runtime.MustLookupBuiltinType("experimental", "to")
 	runtime.ReplacePackageValue("experimental", "to", flux.MustValue(flux.FunctionValueWithSideEffect("to", createToOpSpec, toSignature)))
-	flux.RegisterOpSpec(ExperimentalToKind, func() flux.OperationSpec { return &ToOpSpec{} })
 	plan.RegisterProcedureSpecWithSideEffect(ExperimentalToKind, newToProcedure, ExperimentalToKind)
```

Comment:
doesn't Flux also register a procedure spec for `experimental.to`? I would have thought that registration here would conflict and cause a runtime error?

Reply:
It does. The `ExperimentalToKind` name in this package was changed to avoid that conflict.